Blockzilla/TelemetryIntegration.swift

```diff
@@ -22,6 +22,8 @@ class TelemetryEventMethod {
     public static let open = "open"
     public static let openAppStore = "open_app_store"
     public static let share = "share"
+    public static let navigateBack = "swipe_to_navigate_back"
+    public static let navigateForward = "swipe_to_navigate_forward"
```

Comment:
Please rename these static variables to reflect what they actually refer to. i.e. `swipeToNavigateBack` and `swipeToNavigateForward`